client/src/utils/client-refresh.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: Add lint rule for undescored vars to avoid this
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function handleClientRefresh<T>(_response: Response): Promise<T> {
+  location.reload();
```

Comment:
I think this should be opt-in by the user. They click a confirmation button or something that forces the reload instead of the automation.